code/game/objects/items/implants/implant_explosive.dm

```diff
@@ -214,6 +216,107 @@
 	instant_explosion = FALSE
 	notify_ghosts = FALSE
 
+/// Implant used by the traitor Battle Royale objective, is not active immediately
+/obj/item/implant/explosive/battle_royale
+	name = "rumble royale implant"
+	actions_types = null
+	instant_explosion = FALSE
+	master_implant = TRUE
+	delay = 10 SECONDS
+	panic_beep_sound = TRUE
+	/// Is this implant active yet?
+	var/battle_started = FALSE
+	/// Are we presently exploding?
+	var/has_exploded = FALSE
+
+/obj/item/implant/explosive/battle_royale/get_data()
+	return "<b>Implant Specifications:</b><BR> \
+		<b>Name:</b> Donk Co. 'Rumble Royale' Contestant Motivation Implant<BR> \
+		<b>Life:</b> Activates upon death, or expiry of an internal timer.<BR> \
+		<b>Important Notes:</b> Explodes.<BR> \
+		<HR> \
+		<b>Implant Details:</b><BR> \
+		<b>Function:</b> Contains a compact, electrically detonated explosive that detonates upon receiving a specially encoded signal or upon host death. \
+		Upon triggering the timer, the implant will begin to broadcast the surrounding area for the purposes of televised entertainment. This signal can be detected by GPS trackers.<BR> \
+		<b>Special Features:</b> Exploding.<BR>"
+
+/obj/item/implant/explosive/battle_royale/on_death(datum/source, gibbed)
+	if (!battle_started)
+		return
+	return ..()
+
+/obj/item/implant/explosive/battle_royale/implant(mob/living/target, mob/user, silent, force)
+	. = ..()
+	if (!.)
+		return
+	RegisterSignal(target, COMSIG_LIVING_LIFE, PROC_REF(on_life))
+	if (!battle_started)
+		return
+	name = "[name] - [imp_in.real_name]"
+	target.AddComponent( \
+		/datum/component/simple_bodycam, \
+		camera_name = "rumble royale tracker", \
+		c_tag = "Competitor [target.real_name]", \
+		network = BATTLE_ROYALE_CAMERA_NET, \
+		emp_proof = TRUE, \
+	)
+	announce()
+
+/obj/item/implant/explosive/battle_royale/removed(mob/target, silent, special)
+	. = ..()
+	UnregisterSignal(target, COMSIG_LIVING_LIFE)
+	if (battle_started)
+		qdel(target.GetComponent(/datum/component/simple_bodycam)) // Surely there's no way they could have more than one
+	visible_message(span_boldwarning("[src] beeps ominously."))
+	playsound(loc, 'sound/items/timer.ogg', 50, vary = FALSE)
+
+	if (has_exploded || QDELETED(src))
+		return
+	if (prob(50))
+		explode()
+	else
+		timed_explosion()
+
+/obj/item/implant/explosive/battle_royale/explode()
+	has_exploded = TRUE
+	return ..()
+
+/// Give a slight tell
+/obj/item/implant/explosive/battle_royale/proc/on_life(mob/living/source)
+	SIGNAL_HANDLER
+	if (prob(98))
+		return
+	to_chat(source, span_danger("You scratch at yourself."))
```

Comment:
(IMO) This should match an existing "itch" message we have, like the one the itching viral symptom gives, to throw people off / paranoia / etc. 

The other one is fine, though. The idea being the itch makes them feel paranoid and the lump can confirm it?

Reply:
Unfortunately the other two places with itching both handle it differently to each other as well, though I could try and standardise it in some way. I'm a little wary of doing so without inadvertently causing a balance change... but probably itching powder and itching symptoms are not a huge pain point.